trento/xml/article_sap_trento.xml

```diff
@@ -2309,13 +2310,23 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
 
       <section>
         <title>Trento Support Plugin</title>
-        <para> The Trento Support Plugin automates the collection of relevant
-          runtime information. To use it, proceed as follows: </para>
+        <para> The Trento Support Plugin automates the collection of logs and relevant
+          runtime information on the server side. Using the plugin requires a host 
+         with the following setup:</para>
+                   <itemizedlist>
+          <listitem>
+            <para>Packages jq and python3-yq are installed.</para>
+          </listitem>
+          <listitem>
+            <para>helm is installed;.</para>
+          </listitem>
+          <listitem>
+            <para>kubectl is installed and connected to the K8s cluster where Trento Server is running.
```

Comment:
I think, we should amend the sentence about what this "kubectl" thing is. If I'm not mistaken, it's a command so:

```suggestion
            <para>The command <command>kubectl</command> is installed and connected to the K8s cluster where Trento Server is running.
```

Reply:
I'm not sure about this one. kubectl is more a tool to manage K8s clusters than a command.

Reply:
Really? I found it with the `cnf` (command not found):

```
$ $ cnf kubectl
                     
The program 'kubectl' can be found in following packages:
  * kubernetes1.17-client [ path: /usr/bin/kubectl, repository: zypp (openSUSE-Leap-15.5-1) ]
  * kubernetes1.18-client [ path: /usr/bin/kubectl, repository: zypp (openSUSE-Leap-15.5-1) ]
  * kubernetes1.19-client [ path: /usr/bin/kubectl, repository: zypp (openSUSE-Leap-15.5-1) ]
  * kubernetes1.20-client [ path: /usr/bin/kubectl, repository: zypp (openSUSE-Leap-15.5-1) ]
  * kubernetes1.23-client [ path: /usr/bin/kubectl, repository: zypp (openSUSE-Leap-15.5-1) ]
  * kubernetes1.24-client [ path: /usr/bin/kubectl, repository: zypp (openSUSE-Leap-15.5-1) ]
  * kubernetes1.17-client [ path: /usr/bin/kubectl, repository: zypp (repo-oss) ]
  * kubernetes1.18-client [ path: /usr/bin/kubectl, repository: zypp (repo-oss) ]
  * kubernetes1.19-client [ path: /usr/bin/kubectl, repository: zypp (repo-oss) ]
  * kubernetes1.20-client [ path: /usr/bin/kubectl, repository: zypp (repo-oss) ]
  * kubernetes1.23-client [ path: /usr/bin/kubectl, repository: zypp (repo-oss) ]
  * kubernetes1.24-client [ path: /usr/bin/kubectl, repository: zypp (repo-oss) ]
  * kubernetes1.18-client [ path: /usr/bin/kubectl, repository: zypp (repo-sle-update) ]
  * kubernetes1.23-client [ path: /usr/bin/kubectl, repository: zypp (repo-sle-update) ]
  * kubernetes1.24-client [ path: /usr/bin/kubectl, repository: zypp (repo-sle-update) ]

To install, try:
    sudo zypper install <selected_Package>
```

So `kubectl` is a command that you can execute, hence my suggestion. :slightly_smiling_face:

Reply:
OK, in that case let's leave it as a command.

Reply:
If you want to avoid command, we could also refer to the package name. However, that raises the problem that there are more possible variations of the package name (as it's shown above in the result). Perhaps we could mitigate it by writing something like:

    The package kubernetes<VERSION>-client is installed and connnected...

Hence, I thought it would be easier to just refer to the command instead of the package.

Reply:
No, no, that's too complicated. Let's leave it as a command.